src/Message/ButtonReplyMessage.php

```diff
@@ -3,20 +3,21 @@
 namespace Netflie\WhatsAppCloudApi\Message;
 
 use Netflie\WhatsAppCloudApi\Message\ButtonReply\ButtonAction;
+use Netflie\WhatsAppCloudApi\Message\ButtonReply\Header;
 
-class ButtonReplyMessage extends Message
+final class ButtonReplyMessage extends Message
```

Comment:
The class `ButtonReplyMEssage` yet exists in previous versions so you cannot convert it to final class because this is a breaking change (if anyone extended this class in our code you'll be breaking it)